modules/puppet/manifests/master/package.pp

```diff
@@ -34,11 +30,6 @@
     provider => system_gem,
   }
 
-  package { 'sentry-raven':
-    ensure   => '2.2.0',
-    provider => 'system_gem',
-  }
```

Comment:
Similar to my other comment, this will first need to be `ensure => absent` to make sure it gets uninstalled.